Cargo.toml

```diff
@@ -215,10 +215,10 @@ icu_benchmark_macros = { path = "tools/benchmark/macros" }
 
 # The version here can either be a `version = ".."` spec or `git = "https://github.com/rust-diplomat/diplomat", rev = ".."`
 # Diplomat must be published preceding a new ICU4X release but may use git versions in between
-diplomat = { git = "https://github.com/Manishearth/diplomat", rev = "0fd371d03cb11866a87782762e3fd445e13a3800", default-features = false }
-diplomat-runtime = { git = "https://github.com/Manishearth/diplomat", rev = "0fd371d03cb11866a87782762e3fd445e13a3800", default-features = false }
-diplomat_core = { git = "https://github.com/Manishearth/diplomat", rev = "0fd371d03cb11866a87782762e3fd445e13a3800", default-features = false }
-diplomat-tool = { git = "https://github.com/Manishearth/diplomat", rev = "0fd371d03cb11866a87782762e3fd445e13a3800", default-features = false }
+diplomat = { git = "https://github.com/Manishearth/diplomat", rev = "9739cb1a2cb615c930cafce52a02aa465a50e293", default-features = false }
```

Comment:
issue: this should point to the rust-diplomat repo

Reply:
Oops, my bad!